package.json

```diff
@@ -96,6 +96,7 @@
     "pdfjs-dist": "2.12.313",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-router-dom": "6.3.0",
```

Comment:
Tested in webOS 1.2 emulator: `const` in `main.bundle` from `react-router-dom`.
We need to babel it for legacy browsers.